packages-exp/performance-exp/src/services/transport_service.ts

```diff
@@ -65,6 +66,10 @@ export function setupTransportService(): void {
   }
 }
 
+export function readQueue(): BatchEvent[] {
```

Comment:
Let's avoid exposing the internals of the service just for testing.

Reply:
I agree, I removed this exposure, and validate the `fetch` calls with expected parameter instead.